src/conversions/smallvec.rs

```diff
@@ -54,6 +56,27 @@ where
     }
 }
 
+impl<'py, A> IntoPyObject<'py> for SmallVec<A>
+where
+    A: Array,
+    A::Item: IntoPyObject<'py>,
+    PyErr: From<<A::Item as IntoPyObject<'py>>::Error>,
+{
+    type Target = PyList;
```

Comment:
Possible question if this will become `PyAny` for `SmallVec<u8>` creating bytes.